### ISSUE TITLE
fix: 🤔 Tag inside disabled Combobox is not disabled

### ISSUE THREAD

**Issue by auroraVasconcelos** [solid-design-system/solid#2735]
## Current behavior
When the Combobox is set to disabled, the tag (sd-tag) rendered inside the combobox does not receive the expected disabled state.
As a result, the tag appears and behaves as if it were still enabled, even though the parent combobox is disabled.

## Expected behavior
When the Combobox is disabled, any tag (sd-tag) rendered inside it should also be disabled, visually and functionally, to reflect the disabled state of the component.

## Steps to reproduce
Go to the Combobox screenshot test “Disabled x Multiple”. [Link](https://solid-design-system.fe.union-investment.de/docs/?path=/story/components-sd-combobox-screenshots-sd-combobox--disabled-multiple)
Observe the rendered tags inside the disabled combobox.

## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

**Comment by martascorreia** [comment]
Fixed by this PR: https://github.com/solid-design-system/solid/pull/2751
